### ISSUE TITLE
widely available does not report Promise.try

### ISSUE THREAD

**Issue by ThisIsManta** [3ru/eslint-plugin-baseline-js#114]
According to [MDN `Promise.try()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/try#browser_compatibility) is **newly** available, however, when I use this plugin with **Oxlint** v1.60.0 with the recommended config, it does not report `Promise.try()` in my codebase.

```js
// oxlint.config.js
export default defineConfig({
  options: {
    typeCheck: false,
    typeAware: true,
  },
  jsPlugins: ["eslint-plugin-baseline-js"],
  rules: {
    "baseline-js/use-baseline": [
      "error",
      {
        "available": "widely",
        "includeWebApis": { "preset": "auto" },
        "includeJsBuiltins": { "preset": "auto" } // 👈 does this detect Promise.try?
      }
    ]
  }
}
```

**Comment by 3ru** [comment]
Thanks for the report! ~I was able to reproduce it and will be releasing a fix soon~

You can keep your settings as is. Please check it again in [`v0.6.2`](https://github.com/3ru/eslint-plugin-baseline-js/releases/tag/v0.6.2) :)
